src/Client/Web/Bit.Client.Web.BlazorUI/Scripts/bit.blazorui.ts

```diff
@@ -71,7 +71,10 @@ class Bit {
 }
 
 window.addEventListener('scroll', (e: any) => {
-    if (e.target.id && Bit.currentDropDownCalloutId === e.target.id) return;
+    const minimumWidthForDropDownNormalOpen = 640;
+    if (window.innerWidth < minimumWidthForDropDownNormalOpen ||
```

Comment:
We must checkout 2 conditions:
In the first condition, we must check if the screen size is lower than 640px and the callout which is open is the drop-down callout, then return.
In the second one, we must check if the scroll event target is the drop-down callout, then return.